Makefile

```diff
@@ -65,8 +65,8 @@ dev-image:
             echo "ERROR: QUAY_USERNAME is not set"; \
             exit 1; \
     fi
-	docker build . -t quay.io/$(QUAY_USERNAME)/hacbs-jvm-controller:dev
-	docker push quay.io/$(QUAY_USERNAME)/hacbs-jvm-controller:dev
+	docker build . -t quay.io/$(QUAY_USERNAME)/hacbs-jvm-controller:"$${JBS_QUAY_IMAGE_TAG:-dev}"
+	docker push quay.io/$(QUAY_USERNAME)/hacbs-jvm-controller:"$${JBS_QUAY_IMAGE_TAG:-dev}"
```

Comment:
Unless the JBS_QUAY_IMAGE_TAG is specified we default to `dev`. With Tekton pipelines, if tag is `latest` the it will use PullAlways. Hence for pipelines setting JBS_QUAY_IMAGE_TAG=latest is advised. We should consider whether to change the default to `latest` (perhaps in another iteration).